submissions/blob-arena.md

```diff
@@ -0,0 +1,32 @@
+# Blob Arena
+
+### Project Summary
+Give a short summary of your game project.
+> Tell us about your game!
+
+### GitHub
+Provide the GitHub where your project is hosted. Please ensure your github repo is registered with [OnlyDust](https://app.onlydust.com/p/create) to receive awards.
+> [Contracts](https://github.com/GrugLikesRocks/Blob-arena/)
+
+### Play
+Provide instructions on how to play your game. It could be a URL, a README, or a link to a binary to install.
+> [README.md](https://github.com/GrugLikesRocks/Blob-arena/edit/main/README.md)
```

Comment:
```suggestion
> [README.md](https://github.com/GrugLikesRocks/Blob-arena)
```

@ben-jay-amin @GrugLikesRocks the link to the game build in the repo readme looks dead. Can we upload the artifacts to github instead? You might need to use LFS